README.md

```diff
@@ -37,6 +37,7 @@
 #### Just GIF'S 👻
 - [Ari](https://github.com/ari-hacks/ari-hacks)
 - [Shanu Mishra](https://github.com/Shanu1515/Shanu1515)
+- [Shubham Kumar](https://github.com/imskr)
```

Comment:
@imskr could you add your repo url instead of GitHub url 
I mean add `https://github.com/imskr/imskr` instead of `https://github.com/imskr`

Reply:
Done